Sphinx make failing because of a syntax error present in file

Sphinx make failing:

```
root@nitin-salt-develop:~$ make html
Running Sphinx v1.3.1
loading translations [en]... done
loading pickled environment... done
Exception occurred:
  File "/usr/lib/python2.6/site-packages/sphinx/ext/autosummary/__init__.py", line 502, in _import_by_name
    __import__(modname)
  File "/root/salt/salt/roster/clustershell.py", line 41
    host_addrs = {h: socket.gethostbyname(h) for h in hosts}
                                               ^
SyntaxError: invalid syntax
The full traceback has been saved in /tmp/sphinx-err-K6Gzqf.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 1
```

Full traceback:
```
root@nitin-salt-develop:~/salt$ cat /tmp/sphinx-err-K6Gzqf.log
# Sphinx version: 1.3.1
# Python version: 2.6.6 (CPython)
# Docutils version: 0.11 release
# Jinja2 version: 2.7.3
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/sphinx/cmdline.py", line 244, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/usr/lib/python2.6/site-packages/sphinx/application.py", line 188, in __init__
    self._init_builder(buildername)
  File "/usr/lib/python2.6/site-packages/sphinx/application.py", line 250, in _init_builder
    self.emit('builder-inited')
  File "/usr/lib/python2.6/site-packages/sphinx/application.py", line 497, in emit
    results.append(callback(self, *args))
  File "/usr/lib/python2.6/site-packages/sphinx/ext/autosummary/__init__.py", line 564, in process_generate_options
    base_path=app.srcdir)
  File "/usr/lib/python2.6/site-packages/sphinx/ext/autosummary/generate.py", line 131, in generate_autosummary_docs
    name, obj, parent, mod_name = import_by_name(name)
  File "/usr/lib/python2.6/site-packages/sphinx/ext/autosummary/__init__.py", line 473, in import_by_name
    obj, parent, modname = _import_by_name(prefixed_name)
  File "/usr/lib/python2.6/site-packages/sphinx/ext/autosummary/__init__.py", line 502, in _import_by_name
    __import__(modname)
  File "/root/salt/salt/roster/clustershell.py", line 41
    host_addrs = {h: socket.gethostbyname(h) for h in hosts}
                                               ^
SyntaxError: invalid syntax
```

--- a/salt/roster/clustershell.py
+++ b/salt/roster/clustershell.py
@@ -38,7 +38,7 @@ def targets(tgt, tgt_type='glob', **kwargs):
         ports = list(map(int, str(ports).split(',')))
 
     hosts = list(NodeSet(tgt))
-    host_addrs = {h: socket.gethostbyname(h) for h in hosts}
+    host_addrs = dict([(h, socket.gethostbyname(h)) for h in hosts])
 
     for host, addr in host_addrs.items():
         addr = str(addr)

